tough/src/transport.rs

```diff
@@ -1,8 +1,9 @@
 #[cfg(feature = "http")]
-use crate::{ClientSettings, HttpTransport};
+use crate::{HttpTransport, HttpTransportBuilder};
 use dyn_clone::DynClone;
-use snafu::Snafu;
-use std::fmt::Debug;
+use serde::export::Formatter;
+use std::error::Error;
+use std::fmt::{Debug, Display};
```

Comment:
```suggestion
use std::error::Error;
use std::fmt::{Debug, Display, Formatter};
```
Serde says `export` is `Not public API.` even though it's marked `pub`.  I think it's just a reexport of `std::fmt::Formatter`, though?

Reply:
oh my IDE failed to choose the right thing.